packages/react-composites/src/composites/CallComposite/components/LocalAndRemotePIP.tsx

```diff
@@ -102,31 +104,37 @@ export const LocalAndRemotePIP = (props: LocalAndRemotePIPProps): JSX.Element =>
     [ariaLabel]
   );
 
+  /* @conditional-compile-remove(rooms) */
+  const permissions = _usePermissions();
+  let canTurnCameraOn = true;
+  /* @conditional-compile-remove(rooms) */
+  canTurnCameraOn = permissions.cameraButton;
+
   // If there are no remote participants, show the local participant as the primary tile
   const primaryTileProps: _PictureInPictureInPictureTileProps = useMemo(
     () => ({
       children: remoteVideoTileProps ? (
         <_RemoteVideoTile {...remoteVideoTileProps} />
       ) : (
-        <_LocalVideoTile {...localVideoTileProps} />
+        canTurnCameraOn && <_LocalVideoTile {...localVideoTileProps} />
```

Comment:
wont this render just a white square in the PIPIP?

Reply:
Good catch! Moved the changes to the parent, ModalLocalAndRemotePIP,